defaults/main.yml

```diff
@@ -24,6 +24,9 @@ varnishd_extra_options: ""
 varnish_enabled_services:
   - varnish
 
+# default to a lower priority than Amazon Linux default repos:
+varnish_packagecloud_repo_yum_repository_priority: "9"
```

Comment:
Why not just set it to `1` since, if you don't want to use this role's repo for Varnish install, it would be better to skip adding it altogether?

Reply:
BTW, this PR is set to `[☑︎] Allow edits from maintainers.`, so feel free to push edits that make you comfortable.

If priority of `1` works for you, it works for me.

I've updated the PR description with more details (like "priority numbers closer to `1` have higher priority")

Re:
> if you don't want to use this role's repo for Varnish install,

I see what you mean.  `1` works.